test/JDBC/input/PARTITION-vu-prepare.mix

```diff
@@ -1161,4 +1163,45 @@ GO
 CREATE VIEW SysSearchPartitionNULLDepView
 AS 
     SELECT sys.search_partition('IntPartitionFunction', 10, NULL)
+GO
+
+--------------------------------------------------
+--- To test Collation of Partition Functions
+--------------------------------------------------
+
+USE PartitionDb;
+GO
+
+-- Create partition functions without explicit collation (will use database default)
```

Comment:
Let's add tests for the following:
1. Create partition functions with explicit collation
2. creating table with NVARCHAR column against VARCHAR partition function and vice versa
     a. With same collation
     b. with different collations

Reply:
1. I have already added tests in test/JDBC/input/PARTITION-collations.sql for all possible scenarios.
2. We have already such tests present [here](https://github.com/amazon-aurora/babelfish_extensions/blob/047b8bf62292db0cf63f206b3590b0b0ea2bf069/test/JDBC/expected/PARTITION-vu-verify.out#L2888).